tests/test_client.py

```diff
@@ -184,6 +184,28 @@ def test_authorization(self):
         expected = "Basic " + b64encode(urllib.parse.unquote(login_password)).strip()
         self.assertEqual(expected, headers.get("Authorization"))
 
+    def test_localization_header_default(self):
+        """Localization header not passed to server without explicitly setting SG localization config to True"""
+        self.sg.info()
+
+        args, _ = self.sg._http_request.call_args
+        (_, _, _, headers) = args
+        expected_header_value = "auto"
+
+        self.assertEqual(None, headers.get("locale"))
+
+    def test_localization_header_when_localized(self):
+        """Localization header passed to server when setting SG localization config to True"""
+        self.sg.config.localized = True
+
+        self.sg.info()
+
+        args, _ = self.sg._http_request.call_args
+        (_, _, _, headers) = args
+        expected_header_value = "auto"
+
+        self.assertEqual("auto", headers.get("locale"))
```

Comment:
I think we probably should test whether the schema content coming back from the server is actually localised. Is this easy to do?

Reply:
It's not hard to do (not sure if we can set the site language from the API, though) but then it would test the server, not the client.
IMO if the requests is made with the right header, the job is done.
Also, the localization on the server is already tested with Cypress on top of Ruby unit tests.

Reply:
that makes sense! I was thinking about testing the "receiving end" of the transport, eg. asserting that the received object is a `str` and that it's utf-8 encoded etc. Maybe this is one for @willis102 to weigh in on - do you think this test would be meaningful in a py3 cpntext?